api/tests/acceptance/application/authentication/oidc/token-route-post_test.js

```diff
@@ -151,6 +151,67 @@ describe('Acceptance | Route | oidc | token', function () {
       expect(response.result['logout_url_uuid']).to.match(uuidPattern);
     });
 
+    context('when the PE API does not respond within timeout', function () {
```

Comment:
Suggestion : on essaye de rendre le code oidc générique

```suggestion
context('when the identity provider API does not respond within timeout', function () {

Reply:
Done